test/poolFixtures.hpp

```diff
@@ -456,4 +456,30 @@ TEST_P(umfPoolTest, allocMaxSize) {
     ASSERT_EQ(ptr, nullptr);
 }
 
+#if defined(__has_feature)
+#if __has_feature(address_sanitizer)
+#define __SANITIZE_ADDRESS__
+#endif
+#endif
+
+#ifndef __SANITIZE_ADDRESS__
+// Sanitizer replaces malloc_usable_size implementation with it's own,
+// what results in a failure.
+
+TEST_P(umfPoolTest, mallocUsableSize) {
+    for (size_t allocSize : {32, 48, 1024, 8192}) {
+        char *ptr = static_cast<char *>(umfPoolMalloc(pool.get(), allocSize));
```

Comment:
why cast to `char *`?

Reply:
I just wanted to have a byte array there. Would you like `uint8_t` better, or maybe some wider type?

Reply:
no casting? just `void *`?

Reply:
But how to write then?